mleap-xgboost-runtime/src/main/scala/ml/combust/mleap/xgboost/runtime/bundle/ops/XGBoostClassificationOp.scala

```diff
@@ -23,8 +23,10 @@ class XGBoostClassificationOp extends MleapOp[XGBoostClassification, XGBoostClas
                       (implicit context: BundleContext[MleapContext]): Model = {
       val out = Files.newOutputStream(context.file("xgboost.model"))
       obj.booster.saveModel(out)
-      model.withValue("num_features", Value.int(obj.numFeatures)).
-        withValue("num_classes", Value.int(obj.numClasses))
+      model
+        .withValue("num_features", Value.int(obj.numFeatures))
+        .withValue("num_classes", Value.int(obj.numClasses))
+        .withValue("tree_limit", Value.int(obj.treeLimit))
```

Comment:
Was treeLimit not being saved earlier, and this was a bug. If so, I think we should report it as a bug, and mention that bug here. It will help folks facing this issue!

Reply:
Serializing a model using this Op makes it unusable because the `load()` expects a `tree_limit` and it blows up when it's not there. But I am not sure if anyone stores a Runtime model? 

In any case, I opened an issue here https://github.com/combust/mleap/issues/639